lib/pbench/agent/tool_meister.py

```diff
@@ -295,6 +377,7 @@ def __init__(self, pbench_bin, params, redis_server, logger):
             self._tools,
         ) = ret_val
         self._running_tools = dict()
+        self._perma_tools = dict()
```

Comment:
Again, I'd prefer `_persistent_tools` just because when we're editing this later variations in spelling make errors easier.

Reply:
Yeah permatools is a now deprecated name from before the concept of persistent tools were finalized. I will change these